### ISSUE TITLE
Run third test migration of production Hydrus content to H2 stage

### ISSUE THREAD

**Issue by mjgiarlo** [sul-dlss/happy-heron#1226]
# Prerequisites

- [x] Ensure `sul-h2-stage:/data/h2-files` has ~5.6T free storage space (for storing H2 uploaded files)
- [x] Ensure `sul-h2-stage:/data/hydrus-files` is mounted (with Hydrus production uploaded files)
- [x] Get updated list of collection druids that have been identified for migrating (I'll use the ones from the second migration)
- [x] Rebase `migration` branch of Hydrus on `main` so that it may be safely deployed to prod for the export
- [x] Rebase `migrations2` branch of H2 on `main` so it may be deployed cleanly

# Subtasks

- [x] Deploy `migration` branch of Hydrus to production
- [x] Create `collection_druids.txt` with collection druids on Hydrus prod (or create it locally and scp it, whatever)
- [x] Run Hydrus exporter on sul-hydrus-prod but only for the specified collection druids (ideally in a `screen` session)
  - [x] [Export collections](https://github.com/sul-dlss/hydrus/pull/507/files#diff-14e7df408fd5c4aef0b86331103bd44e56c2d000c10c535df59db6a9d5cd6982) on hydrus-prod: exported **162** collections with **0** errors
    * Run: `RAILS_ENV=production bin/export-collections > collections.jsonl`
  - [x] Transfer exported collection JSON from hydrus-prod to sul-h2-stage at `~/collections.jsonl`
- [x] Run Hydrus exporter on sul-hydrus-prod but only for items belonging to the specified collection druids (ideally in a `screen` session)
  - [x] [Export items](https://github.com/sul-dlss/hydrus/pull/507/files#diff-6c900ce23a9f4fd849fb84f5f983892e79ac44052b6d2421f08f84d74b97e12e) on hydrus-prod: exported **3,825** items with **0** errors and **1,301** skips ([see below](https://github.com/sul-dlss/happy-heron/issues/1226#issuecomment-786800965))
    * Run: `RAILS_ENV=production bin/export-items > items.jsonl 2> item-errors.log`
  - [x] Transfer exported item JSON from hydrus-prod to sul-h2-stage at `~/items.jsonl`
- [x] Deploy `migrations2` branch of H2 to stage
- [x] Clear stage environment (leaving only users behind)
  * Run: `RAILS_ENV=production bin/rails r 'Work.update_all(head_id: nil)' && RAILS_ENV=production bin/rails r 'Collection.update_all(head_id: nil)' && RAILS_ENV=production bin/rails r 'Collection.destroy_all' && RAILS_ENV=production bin/rails r 'ContactEmail.destroy_all' && RAILS_ENV=production bin/rails r 'Event.destroy_all' && RAILS_ENV=production bin/rails r 'ActiveStorage::Blob.destroy_all'`
  * Counts (`[AbstractContributor.count, RelatedWork.count, RelatedLink.count, WorkVersion.count, ActiveStorage::Attachment.count, User.count, ActiveStorage::Blob.count, Collection.count, AttachedFile.count, Event.count, ContactEmail.count, Keyword.count, Work.count, CollectionVersion.count]`)
    * Before clearing: `[12687, 681, 1181, 4794, 12201, 1736, 12198, 177, 12201, 33314, 4967, 25462, 4844, 177]`
    * After clearing: `[0, 0, 0, 0, 0, 1736, 0, 0, 0, 0, 0, 0, 0, 0]` 
- [x] [Import collections](https://github.com/sul-dlss/happy-heron/pull/777/files) on sul-h2-stage
  * Run:
    * `RAILS_ENV=production bin/rails r 'puts "Users: #{User.count}\nContactEmails: #{ContactEmail.count}\nEvents: #{Event.count}\nRelatedLinks: #{RelatedLink.count}\nCollections: #{Collection.count}\nCollectionVersions: #{CollectionVersion.count}"' | tee -a migrate-collections.log`
    * `RAILS_ENV=production bin/migrate-collections ~/collections.jsonl 2>&1 | tee -a migrate-collections.log`
    * `RAILS_ENV=production bin/rails r 'puts "Users: #{User.count}\nContactEmails: #{ContactEmail.count}\nEvents: #{Event.count}\nRelatedLinks: #{RelatedLink.count}\nCollections: #{Collection.count}\nCollectionVersions: #{CollectionVersion.count}"' | tee -a migrate-collections.log`
  * Results:
    ```
    BEFORE

    Users: 1736
    ContactEmails: 0
    Events: 0
    RelatedLinks: 0
    Collections: 0
    CollectionVersions: 0

    AFTER

    Users: 1736
    ContactEmails: 164
    Events: 1744
    RelatedLinks: 158
    Collections: 161
    CollectionVersions: 161
    ```
- [x] [Import items and files](https://github.com/sul-dlss/happy-heron/pull/777/files) on sul-h2-stage: **Copy/paste results in a comment below**
  * Run: 
    * `RAILS_ENV=production bin/rails r 'puts "Users: #{User.count}\nAuthors: #{Author.count}\nContributors: #{Contributor.count}\nContactEmails: #{ContactEmail.count}\nEvents: #{Event.count}\nKeywords: #{Keyword.count}\nRelatedLinks: #{RelatedLink.count}\nRelatedWorks: #{RelatedWork.count}\nWorks: #{Work.count}\nWorkVersions: #{WorkVersion.count}"' | tee -a migrate-items.log`
    * `RAILS_ENV=production bin/migrate-items ~/items.jsonl 0 /data/hydrus-files 2>&1 | tee -a migrate-items.log`
    * `RAILS_ENV=production bin/rails r 'puts "Users: #{User.count}\nContributors: #{Contributor.count}\nContactEmails: #{ContactEmail.count}\nEvents: #{Event.count}\nKeywords: #{Keyword.count}\nRelatedLinks: #{RelatedLink.count}\nRelatedWorks: #{RelatedWork.count}\nWorks: #{Work.count}\nWorkVersions: #{WorkVersion.count}"' | tee -a migrate-items.log`
  * Results:
    ```
    BEFORE

    Users: 1736
    Authors: 0
    Contributors: 0
    ContactEmails: 164
    Events: 1744
    Keywords: 0
    RelatedLinks: 158
    RelatedWorks: 0
    Works: 0
    WorkVersions: 0

    AFTER

    Users: 1736
    Contributors: 1733
    ContactEmails: 3954
    Events: 27961
    Keywords: 22242
    RelatedLinks: 1101
    RelatedWorks: 684
    Works: 3828
    WorkVersions: 3789
    ```

**Comment by mjgiarlo** [comment]
Skipping druid:kr363kq3288 because it has a catkey                                                                                                    
Skipping druid:yn530ms1593 because it has a catkey                                                                                                    
Skipping druid:bd098th5420 because it has a catkey                                                                                                    
Skipping druid:pv311by5369 because it has a catkey                                                                                                    
Skipping druid:sw721dk1304 because it has a catkey                                                                                                    
Skipping druid:zk127ng8952 because it has a catkey                                                                                                    
Skipping druid:vy553qs2473 because it has a catkey                                                                                                    
Skipping druid:vm464wn7735 because it has a catkey                                                                                                    
Skipping druid:gs767mz5966 because it has a catkey                                                                                                    
Skipping druid:wn642xs9389 because it has a catkey                                                                                                    
Skipping druid:vy083rm9574 because it has a catkey                                                                                                    
Skipping druid:xr651kd7460 because it has a catkey                                                                                                    
Skipping druid:cd663nx5858 because it has a catkey                                                                                                    
Skipping druid:yk203gf1929 because it has a catkey                                                                                                    
Skipping druid:yn069ws1192 because it has a catkey                                                                                                    
Skipping druid:gj570xx5715 because it has a catkey                                                                                                    
Skipping druid:mg392dx5190 because it has a catkey          
Skipping druid:jk795qh0494 because it has a catkey
Skipping druid:fv533hj0048 because it has a catkey
Skipping druid:zy062xv2633 because it has a catkey
Skipping druid:wz110cw1514 because it has a catkey
Skipping druid:hj852yn2610 because it has a catkey
Skipping druid:yh711kg4756 because it has a catkey
Skipping druid:wv446yn2614 because it has a catkey
Skipping druid:rs154fr6978 because it has a catkey
Skipping druid:dy826nt5291 because it has a catkey                                                                                                    
Skipping druid:jk084dk8092 because it has a catkey
Skipping druid:wr034gg3847 because it has a catkey
Skipping druid:zd148dg8878 because it has a catkey
Skipping druid:qj786rq5965 because it has a catkey
Skipping druid:qf102ds5177 because it belongs to the Graveyard APO
Skipping druid:cq239mt9958 because it has a catkey
Skipping druid:wh733rj8799 because it has a catkey
Skipping druid:rm918vk8051 because it has a catkey
Skipping druid:bb815pq7465 because it has a catkey
Skipping druid:qh169pt0560 because it has a catkey
Skipping druid:hy978xk2062 because it has a catkey
Skipping druid:zk481fh3026 because it has a catkey
Skipping druid:nc358hv0923 because it has a catkey
Skipping druid:hh902sy9424 because it has a catkey
Skipping druid:cr541db3254 because it has a catkey                                                                                                    
Skipping druid:sc575my8787 because it has a catkey         
Skipping druid:dm266rq3266 because it has a catkey
Skipping druid:rs803pd3839 because it has a catkey
Skipping druid:nk553yb7192 because it has a catkey
Skipping druid:hf005nn1462 because it has a catkey
Skipping druid:gp554wh1404 because it has a catkey
Skipping druid:by544fp5883 because it has a catkey
Skipping druid:vv639rj3449 because it has a catkey                                                                                                    
Skipping druid:cx492jc0657 because it has a catkey                                                                                                    
Skipping druid:xg667gr4605 because it has a catkey                                                                                                    
Skipping druid:fx090td1295 because it has a catkey                                                                                                    
Skipping druid:gx700ng3717 because it has a catkey                                                                                                    
Skipping druid:yy436vd6426 because it has a catkey                                                                                                    
Skipping druid:ys111hq8598 because it has a catkey                                                                                                    
Skipping druid:qy315fc8502 because it has a catkey                                                                                                    
Skipping druid:pn612ng7865 because it has a catkey                                                                                                    
Skipping druid:xj391jk5611 because it has a catkey                                                                                                    
Skipping druid:hv851tv0209 because it has a catkey
Skipping druid:yx136qb8772 because it has a catkey
Skipping druid:dg715vd8947 because it has a catkey
Skipping druid:yy360cw7391 because it has a catkey
Skipping druid:tv271bx1434 because it has a catkey

**Comment by mjgiarlo** [comment]
# Skips due to item already having catkey

```
druid:bb147tm0649
druid:bb254jh9333
druid:bb654mz9296
druid:bb815pq7465
druid:bb836vc8874
druid:bb881rb8923
druid:bc549qd4424
druid:bc777tp9978
druid:bc859tn2484
druid:bd042mb4183
druid:bd066vp2608
druid:bd098th5420
druid:bd496jj1248
druid:bd646cj8238
druid:bd705nq9853
druid:bf288pt1165
druid:bf426yn5727
druid:bg016hs1268
druid:bg181zw7535
druid:bg474th2148
druid:bg975kv5558
druid:bh284jk9222
druid:bh731xr9048
druid:bh940cz1755
druid:bh997cm9774
druid:bj226kp0465
druid:bj378nr9050
druid:bj615mb0587
druid:bj744zf0960
druid:bj765gm6379
druid:bk433fc3791
druid:bk506kr8679
druid:bk662pt1482
druid:bk689jd2364
druid:bm118jy4627
druid:bm120vp2531
druid:bn726rv0719
druid:bp025mg9274
druid:bp369vt9567
druid:bp720qm3931
druid:bq685wb1563
druid:bq777sx2673
druid:br498vc8942
druid:br667hf8188
druid:br878mj3523
druid:br922mr1816
druid:bs004md7816
druid:bs312hy9655
druid:bs942pt4837
druid:bt690vf9530
druid:bt703qw1328
druid:bt877hj4197
druid:bt964gw1942
druid:bv041vy3567
druid:bv517hw0666
druid:bv748gx5834
druid:bv848kq5382
druid:bw018gd0908
druid:bw636sw1637
druid:bw742dq9478
druid:bx499rs4405
druid:by449tv0181
druid:by520mn9078
druid:by544fp5883
druid:by823qp9435
druid:by838hx3512
druid:bz230kt1480
druid:bz570df3780
druid:bz741tw0638
druid:bz915yj3775
druid:cb014wg7849
druid:cc465tw2838
druid:cc929sv4522
druid:cd121dm4511
druid:cd158rj5479
druid:cd663nx5858
druid:cf341dd3832
druid:cf378cb6321
druid:cf386dp2798
druid:cf610px5268
druid:cf937sd6931
druid:cg330nc6674
druid:cg332hx6079
druid:cg842rd3961
druid:cg987gr7782
druid:ch234sq4216
druid:ch313yw3004
druid:ch328tk0843
druid:ch707km8499
druid:cj626wt2021
druid:cj846pv4746
druid:ck116xd5537
druid:ck152nk0140
druid:ck388yb8844
druid:ck396nz2233
druid:ck609fw9113
druid:ck889yk4686
druid:ck980wq0127
druid:cm084sd0967
druid:cm318pb1689
druid:cm589sz6619
druid:cm749vb6952
druid:cm882rg4981
druid:cm931mp4061
druid:cn001ss1272
druid:cn204tg9894
druid:cn220pm0721
druid:cn394vc0241
druid:cn524mq4250
druid:cn995my3723
druid:cp103jm7836
druid:cp258kh2394
druid:cp557sy8272
druid:cp821fx1106
druid:cp909yz1729
druid:cq075dm0981
druid:cq239mt9958
druid:cq730zc6783
druid:cq913sm6547
druid:cq916xj7053
druid:cr541db3254
druid:cr608dp4894
druid:cr987rv2748
druid:cs091sn3803
druid:cs269vf5063
druid:cs439sn7374
druid:cs826pj6923
druid:ct462nd4172
druid:ct608qw0779
druid:ct719py2618
druid:ct851kh2715
druid:cv123sz0279
druid:cv140wy2090
druid:cv601tz9723
druid:cw515yv2052
druid:cx273zg8480
druid:cx464sh3548
druid:cx492jc0657
druid:cx997pr0242
druid:cy080rh3308
druid:cy671wq8928
druid:cy687jf5081
druid:cy714yp3149
druid:db055qp6447
druid:db138yt9617
druid:db179cg3223
druid:db200qs5339
druid:db467zf5938
druid:db554sv4088
druid:dc757cy6358
druid:dc872vf7340
druid:dc938tw1522
druid:dd304hw6904
druid:dd316mt6216
druid:dd523hz7568
druid:dd647ns9954
druid:dd770dn3247
druid:dd953yp2986
druid:dd995vq1826
druid:df321xy2402
druid:df432dz9486
druid:df484cs8101
druid:df609vr8556
druid:df990dz1752
druid:dg025hs2381
druid:dg100mm4995
druid:dg252cf6401
druid:dg552pb6632
druid:dg715vd8947
druid:dh484hp6955
druid:dh569hj4571
druid:dh727mg8916
druid:dj312bw8688
druid:dj829ch9346
druid:dk221pr7942
druid:dk343jb2070
druid:dk407tb1290
druid:dk802fs5284
druid:dk909sr5708
druid:dm111cx0620
druid:dm266rq3266
druid:dm484wx3131
druid:dn302tv1768
druid:dn535hv1906
druid:dn559yy2014
druid:dn928fx1060
druid:dn941wp4376
druid:dp032wv8092
druid:dp143dx0300
druid:dp372fb6449
druid:dp652sc4970
druid:dp881rx9760
druid:dq314qv7705
druid:dq366zd7945
druid:dq727vg7765
druid:dq879bd1291
druid:dr236gf6554
druid:dr329hp4099
druid:dr445vb3882
druid:dr530wn2701
druid:dr541zr1871
druid:dr738rr4094
druid:dr995dr9669
druid:ds036zh8754
druid:ds982xr8543
druid:dt107ry6700
druid:dt341fk4310
druid:dt691pq8752
druid:dt766bd8775
druid:dv111mq4234
druid:dv124cn2640
druid:dv485bs9648
druid:dw263tg5791
druid:dw377pj9298
druid:dw463rh9043
druid:dx155xq7545
druid:dx534px5545
druid:dx637fr5545
druid:dx809gy3686
druid:dx921sy4228
druid:dy287dx7346
druid:dy740hx0996
druid:dy826nt5291
druid:dy878pd9634
druid:dz450pr2075
druid:dz458qj4164
druid:dz465bp9692
druid:dz613cw1711
druid:dz787mm0890
druid:dz864rz8489
druid:fc505rt4767
druid:fd426tp0728
druid:fd751kw7862
druid:fd955gh8707
druid:ff099sb4520
druid:ff116jw9869
druid:ff499sp0937
druid:ff744zg9965
druid:ff755qy9757
druid:ff945vf5421
druid:fg289mp6421
druid:fg291kw5907
druid:fg330jw9608
druid:fh029yr7753
druid:fh100gd4098
druid:fh210dw0924
druid:fh322sx7093
druid:fh522yt2409
druid:fh556sd9761
druid:fh867bd2918
druid:fh950cq3642
druid:fj130fm7415
druid:fj256wb4874
druid:fj522by6951
druid:fj585ft0580
druid:fj804zs3198
druid:fj877gf8960
druid:fk175qx8156
druid:fm307zc6685
druid:fm392np1292
druid:fm395ry6187
druid:fm709wv7029
druid:fm774hx8787
druid:fn301wc3237
druid:fn347js8478
druid:fn431st6199
druid:fn506sp1649
druid:fp224tk2926
druid:fp279cd4770
druid:fp471rw8580
druid:fp543qd4724
druid:fp559dr7879
druid:fp613vt7240
druid:fp645dw8083
druid:fq474rp3444
druid:fq524yp0934
druid:fq532bc3844
druid:fq616rk5637
druid:fq626hb1300
druid:fr297wg7194
druid:fr418qg4826
druid:fr514hp9379
druid:fr604hf8956
druid:fs692vv4982
druid:ft055yy4350
druid:ft929jk8905
druid:fv216yf4783
druid:fv323zz2024
druid:fv328df9311
druid:fv354yr2874
druid:fv434vc1815
druid:fv533hj0048
druid:fv784pr2639
druid:fv806bb5208
druid:fv961wq0736
druid:fw068kq9871
druid:fw232sm7686
druid:fw535rs1323
druid:fx090td1295
druid:fx208qx5006
druid:fx224yr0086
druid:fx367kc4026
druid:fx738jb6722
druid:fy073vy0675
druid:fy535yb8979
druid:fy768ck1275
druid:fz381vv6581
druid:gb129dk1428
druid:gb133hc5348
druid:gb348ww2246
druid:gb532wv0586
druid:gb553yc6585
druid:gb948cd0244
druid:gc192dv5791
druid:gc774px0892
druid:gd198rv4995
druid:gd222pz0166
druid:gd234sr2440
druid:gd247jy0014
druid:gd562sq0953
druid:gf191xw3607
druid:gf414wr0054
druid:gg008cs3831
druid:gg044th2990
druid:gg413zp1930
druid:gh052qj3575
druid:gj150vd0498
druid:gj244sd9273
druid:gj570xx5715
druid:gm235bn2492
druid:gm673ct4465
druid:gn707jx2482
druid:gn990nb9782
druid:gp257mc2364
druid:gp554wh1404
druid:gq442mg9885
druid:gq864gg8031
druid:gr129xh9595
druid:gr256sv7513
druid:gr468rq4545
druid:gs618wg5492
druid:gs767mz5966
druid:gt217kt6500
druid:gt434zz5968
druid:gv018yn2912
druid:gv052wp3508
druid:gv150dj3496
druid:gv326mh0501
druid:gw182ss7068
druid:gw276hk5940
druid:gw297kq8317
druid:gx700ng3717
druid:gx774gj4320
druid:gy195ry1856
druid:gy963md6289
druid:gy981zj6014
druid:gz247zx1039
druid:gz658mw3117
druid:hb137bm0691
druid:hb361md7468
druid:hb840qf1564
druid:hb970vw9806
druid:hc325zj5890
druid:hd352vk4378
druid:hd564hz7107
druid:hd605dv8585
druid:hd679gb2691
druid:hd872br1568
druid:hf005nn1462
druid:hf580sz7908
druid:hg004rc2530
druid:hg074bf6358
druid:hg277vq4016
druid:hg393qj9453
druid:hg796qd2166
druid:hg862pd5794
druid:hh487yv5503
druid:hh504xb4109
druid:hh658mj2353
druid:hh830jj0377
druid:hh902sy9424
druid:hj151vq9137
druid:hj184fn4277
druid:hj312xm0363
druid:hj368qv6949
druid:hj432tn5471
druid:hj523wr3973
druid:hj719by4965
druid:hj840gy6818
druid:hj852yn2610
druid:hk173rw0887
druid:hk487kq7006
druid:hk813mf1038
druid:hm496cc7367
druid:hn083sc0017
druid:hn620rn0994
druid:hp150sd3241
druid:hp264vj6959
druid:hp434qb8807
druid:hp711dq5781
druid:hp828yz7881
druid:hp846qn0734
druid:hp908jn9255
druid:hq673px9007
druid:hs133tn8779
druid:hs154xr5794
druid:hs308rc3995
druid:hs706fh8622
druid:hv225pz7783
druid:hv697qp3971
druid:hv712gs7377
druid:hv851tv0209
druid:hw205zp5227
druid:hw668kc6604
druid:hx005ms1753
druid:hx104rr9991
druid:hx383pk6976
druid:hx711kn2021
druid:hx735vm3872
druid:hy317hn3813
druid:hy459js4845
druid:hy640yc8117
druid:hy978xk2062
druid:hz166yw2425
druid:hz343qz3924
druid:hz385qn6485
druid:hz835qj2045
druid:jb437yg3324
druid:jb745pf9118
druid:jb896gs6625
druid:jc147nb0924
druid:jd112fp7867
druid:jd289bx2999
druid:jd500fc0475
druid:jd700jx6053
druid:jf303tn2013
druid:jf597pw5053
druid:jg253jk2516
druid:jh019mp0167
druid:jh271rr2079
druid:jh889jc1124
druid:jk084dk8092
druid:jk374sg3280
druid:jk795qh0494
druid:jm140hy6689
druid:jm145vg1898
druid:jm434hr1072
druid:jm939dc5172
druid:jn435wv2825
druid:jn609rs1073
druid:jp040pb4069
druid:jp138js9962
druid:jp416yk3562
druid:jp875sz6810
druid:jp963mr4517
druid:jq221dh9354
druid:jq464gs6934
druid:jq781pn7618
druid:jr239sd0406
druid:jr260mj2846
druid:jr322vm6204
druid:jr324qc6145
druid:jr481xk9183
druid:jr484yj8148
druid:jr542vk0229
druid:js273yj1964
druid:js378qm0633
druid:js527rh4748
druid:js566kc0584
druid:js928tt2355
druid:jt132pw4453
druid:jt438fh5468
druid:jt496kf1245
druid:jt514ks5365
druid:jt546qd0941
druid:jv028tr4263
druid:jv351cy8591
druid:jv666fy2208
druid:jv967qq2258
druid:jw286nj4366
druid:jw362pm6923
druid:jw623hj9094
druid:jx559xh9522
druid:jy189qp0098
druid:jy275mr7931
druid:jy305wh2817
druid:jy604bd1446
druid:jy937pc4379
druid:jy970wp1819
druid:jz208np2893
druid:jz727wy8218
druid:jz926cp5015
druid:jz976gh2740
druid:kb133dc7896
druid:kb235gc1692
druid:kb237cf9161
druid:kc244gd5834
druid:kc689rv7123
druid:kc836jy8410
druid:kc956fr2039
druid:kd771pj9391
druid:kd964cr9650
druid:kf023qf9371
druid:kf532yh6869
druid:kf624fn4367
druid:kf916qc8341
druid:kf999gv4507
druid:kg357gp5418
druid:kg779tp2951
druid:kh054sv7857
druid:kh332fw4507
druid:kh549yh0232
druid:kj049hk9559
druid:kj325kb7215
druid:kj379kj3758
druid:kj943ft2232
druid:kk125tz9132
druid:kk240yg5287
druid:kk290gt9178
druid:kk920js6936
druid:km258pz4280
druid:km508vt5918
druid:km632kp1976
druid:km845dr0603
druid:km987cn8848
druid:kn053dq2621
druid:kn280cc4680
druid:kn947cf6566
druid:kp364dv8444
druid:kp687bs2524
druid:kq500zz3612
druid:kq668hy1994
druid:kq979cc3345
druid:kr020md4080
druid:kr074bj3299
druid:kr345fx4637
druid:kr359xp2679
druid:kr363kq3288
druid:kr494zn8977
druid:kr656fs2158
druid:kr813qs4832
druid:kr860qf1438
druid:kt095wt7013
druid:kt324rp1821
druid:kt451cg5736
druid:kt630sq8173
druid:kt708sp9550
druid:kv132rx2048
druid:kv387xt2364
druid:kv867xr2894
druid:kw504bw4002
druid:kw657mg1285
druid:kw840rx7361
druid:kx048cg0213
druid:kx180st6850
druid:kx793rr3807
druid:kx977xc4809
druid:ky250ky4478
druid:ky956fk2236
druid:kz243sp4067
druid:kz382tx3543
druid:kz462sg7849
druid:kz959wq5362
druid:mb154jz1702
druid:mb654bp1848
druid:mb672ps5828
druid:mb768dc1016
druid:mb968cj9476
druid:mc124dw3653
druid:mc133hj7925
druid:mc655ny6353
druid:mc860wb3920
druid:md672yg1789
druid:md912bk2459
druid:mf099jp8207
druid:mf319pz1741
druid:mg117gj2129
druid:mg363md3645
druid:mg392dx5190
druid:mg442ns8854
druid:mh100rj5405
druid:mh782dw3510
druid:mj124jm1210
druid:mj299cq9246
druid:mj469mb7644
druid:mj554kv0178
druid:mk880hh9444
druid:mn229ds8766
druid:mp099bp7808
druid:mp234kp2063
druid:mp750sm6808
druid:mq297vm4471
druid:mq389px0511
druid:mq763pr9124
druid:mr085wz2827
druid:mr577xc0960
druid:mr959qg6922
druid:mr991kt4275
druid:ms117jw7756
druid:ms288ns4198
druid:ms357tx3662
druid:ms513bj7840
druid:ms955mj1331
druid:mt817wr0164
druid:mt938np7102
druid:mv033pk7572
druid:mv589qm7746
druid:mv603bd9807
druid:mv741jd2621
druid:mv772dh0125
druid:mv824ck4135
druid:mw351dy6792
druid:mw619zv5462
druid:mw830tk7018
druid:my150wy9764
druid:my482wn9003
druid:my648ky0405
druid:my652qk2615
druid:mz096fk2694
druid:mz176bj1558
druid:mz347gf1104
druid:mz989qb3133
druid:nb146js3291
druid:nb470nj4418
druid:nc020zz6976
druid:nc281tr5462
druid:nc358hv0923
druid:nc526gs7754
druid:nc557zb0638
druid:nd016kz9321
druid:nd355nk2711
druid:nd432sq3899
druid:nd619gk6289
druid:nd917vv0597
druid:nf417dw6456
druid:nf513vh4343
druid:ng014ck4496
druid:ng161qh7958
druid:ng453xc6226
druid:ng540yf2358
druid:nh102zf9361
druid:nh189kp7009
druid:nh569fb8298
druid:nh896mz0093
druid:nj305wd5416
druid:nj524tr6642
druid:nj555dw0935
druid:nj878gz8075
druid:nk199xp1662
druid:nk553yb7192
druid:nk799vp7032
druid:nm137ts0627
druid:nm446rs3041
druid:nm518vg5867
druid:nm963yd9915
druid:nn568sd8519
druid:nn591cb6430
druid:nn679sk5834
druid:np032tf0441
druid:np116nb9646
druid:np898mz3341
druid:nq278mx7978
druid:nq576hd9573
druid:nq613nr5612
druid:nr279gk0781
druid:ns232bt0013
druid:ns321rq9480
druid:ns592gv5110
druid:ns902xy8961
druid:nt044hz7665
druid:nt486ds0873
druid:nt627yd2014
druid:nv113mr0486
druid:nw819wh3735
druid:nx294zy2046
druid:nx327nh0351
druid:ny186xk9723
druid:ny778yh4073
druid:ny999bq5621
druid:pb013td8253
druid:pb171gf0539
druid:pb809cy9630
druid:pb823dp9863
druid:pc551gw6209
druid:pd423yp9184
druid:pf136wk5009
druid:pf141bs6760
druid:pf873tj2515
druid:pg239zj9524
druid:pg398pd1954
druid:ph409yp3080
druid:ph826qz5497
druid:pj321rf0947
druid:pj752zb7029
druid:pk052zg0066
druid:pk114vz0868
druid:pk223vd0642
druid:pk310hf9150
druid:pm912cz2709
druid:pn290dw8062
druid:pn612ng7865
druid:pn679ds1629
druid:pn860ph4218
druid:pp071qb9526
druid:pp680jz4992
druid:pp751ky7071
druid:pp921qt9521
druid:pp963gh4327
druid:pq041vp9121
druid:pq223kp0525
druid:pq270qd2941
druid:pq739st8617
druid:pr445yy3871
druid:pr530wb8176
druid:pr779cm9137
druid:ps489hj9360
druid:pt078qc4174
druid:pt274km1220
druid:pt315wb2504
druid:pt320tk7009
druid:pt789vz1932
druid:pv311by5369
druid:pv375gw6813
druid:pv636yw7598
druid:pv852hs3092
druid:pw063sz1430
druid:pw180nx5120
druid:pw668pc4997
druid:px440rw7108
druid:px561zr4731
druid:px729gz2879
druid:pz383kg5459
druid:pz537wx2380
druid:pz652bn5491
druid:qb103dw9880
druid:qb152vj1554
druid:qb600rs3304
druid:qc438kn3399
druid:qc520gb9892
druid:qd599vf1910
druid:qd745hw1009
druid:qf105xq1910
druid:qf714sk6834
druid:qf970xk6381
druid:qg339yk5817
druid:qg497yx7333
druid:qg806bq1199
druid:qg856ns9562
druid:qg882dn4516
druid:qg938kz5636
druid:qh169pt0560
druid:qh215dz1806
druid:qh366bn2187
druid:qh700qy9278
druid:qh763ry8972
druid:qj528qh2080
druid:qj786rq5965
druid:qk364bw1330
druid:qm141fz4326
druid:qn025tn3677
druid:qn048tt9186
druid:qn293vc3189
druid:qn735fj6639
druid:qn963zb5361
druid:qp552sq6463
druid:qp584cr0305
druid:qp647tv6290
druid:qq072cg9915
druid:qq076yg9924
druid:qq707ft7265
druid:qr036qg6433
druid:qr088db3377
druid:qr228xg8005
druid:qr294rd5902
druid:qr907cy5117
druid:qs328fg0035
druid:qs680bn2010
druid:qt143jt3005
druid:qt161sr9482
druid:qt395bb1230
druid:qt822sc0608
druid:qv115xj5800
druid:qv120td6332
druid:qv766ff8373
druid:qw035rb4682
druid:qw138xw8065
druid:qw183sx2533
druid:qw850cq6931
druid:qx284br0664
druid:qx829zz1679
druid:qx979vh4115
druid:qy164rw6883
druid:qy285zx0917
druid:qy315fc8502
druid:qy402tm9843
druid:qy747qc1915
druid:qy800db3255
druid:qz327py5003
druid:qz409bc6646
druid:qz519xz9931
druid:rb087vd6751
druid:rb822qc8579
druid:rc116xv1261
druid:rc320qk3407
druid:rd401pb6506
druid:rd446qg7815
druid:rd679yc4330
druid:rf451vs8933
druid:rf619nj5030
druid:rg443st4909
druid:rg807hs4203
druid:rh026hm9212
druid:rh712qk7553
druid:rh880tt9705
druid:rm290yj6685
druid:rm414nt5035
druid:rm918vk8051
druid:rp413qn4856
druid:rp576mp1187
druid:rp780dm7255
druid:rq145vz8284
druid:rq232pm0533
druid:rq442tz7174
druid:rq476fs2908
druid:rq852kh7618
druid:rr381vy7757
druid:rr460ck2183
druid:rr519fh3826
druid:rs039sc6223
druid:rs154fr6978
druid:rs803pd3839
druid:rt167dk4264
druid:rt175tg9384
druid:rt326bg3965
druid:rw138vr3300
druid:rw182bx2292
druid:rw606wx4376
druid:rw748wz0758
druid:rx317xq1656
druid:rx436wz0577
druid:rx484fk1613
druid:rx516bp3074
druid:ry028jy7465
druid:ry171sz6990
druid:ry332wk2920
druid:ry446zn3981
druid:ry560bs7115
druid:ry825sb6266
druid:ry934cb7732
druid:ry967fb9637
druid:rz234qk3745
druid:rz440qh6496
druid:rz454zn7587
druid:rz878ht1939
druid:rz986yd7965
druid:sb042dr8484
druid:sb386yr8649
druid:sb552xd6993
druid:sc575my8787
druid:sc607bc3362
druid:sc865zz3888
druid:sd545mx0427
druid:sd616qs6517
druid:sd850yq6488
druid:sf127qd3856
druid:sf220yg5438
druid:sf243ph1538
druid:sg000dr6979
druid:sg164vy8514
druid:sg569yb8447
druid:sg842zq7153
druid:sh250fc5053
druid:sh745nv9013
druid:sh813hr4592
druid:sj264qw3380
druid:sj587pn5903
druid:sj837yz2618
druid:sj902jz5764
druid:sk099mm2666
druid:sk199by8463
druid:sk276tp5765
druid:sm049kb1604
druid:sm095xc9951
druid:sm179px1977
druid:sm776mq8441
druid:sm925hd2158
druid:sn140cb4039
druid:sn229md3325
druid:sn392xs6241
druid:sn422gj3786
druid:sp155vm6775
druid:sp211sk4291
druid:sp359zb4496
druid:sp396vv5239
druid:sp404cx9759
druid:sp665qh5380
druid:sp939rc6334
druid:sq170bc0082
druid:sq766fh7804
druid:sr674dh1703
druid:sr810gq4711
druid:ss184nh6941
druid:st147cp9682
druid:st541cm1185
druid:sv355bs2665
druid:sv516tq4696
druid:sv557yy1973
druid:sv751rs6415
druid:sv984vg2157
druid:sw721dk1304
druid:sx255jr0278
druid:sx402ph6635
druid:sx594yg2017
druid:sx620jd0104
druid:sx783qr4650
druid:sy010zt9483
druid:sy465zx2770
druid:sy576zh1215
druid:sy618bd2946
druid:sy809mg8171
druid:tc037rd6519
druid:tc231wt6718
druid:tc309nr4593
druid:tc489rs3928
druid:tc578qh5928
druid:tc936fp4054
druid:tc975zd2606
druid:td901kx5712
druid:td952sw5960
druid:tf181wy6176
druid:tf362jx0047
druid:tf395mb1255
druid:tf672wz2836
druid:tf860rn9355
druid:tg077fs5092
druid:tg091nd2814
druid:tg482sx4858
druid:tg630wm0832
druid:th204gp6905
druid:tj074mm9826
druid:tj890yr0333
druid:tk572kp3332
druid:tm677hy5042
druid:tm899bg8781
druid:tn418kh5434
druid:tn466td5357
druid:tn528np2842
druid:tn788nf0500
druid:tp179ty6460
druid:tp231wb4805
druid:tp843xz0614
druid:tp905vg2436
druid:tq091xc2452
druid:tq617nk9956
druid:tr302qp3638
druid:tr793xd3959
druid:tr874kv0002
druid:tr949rx8677
druid:ts197hy9480
druid:ts670zm1713
druid:tt874xc9394
druid:tv229zj4090
druid:tv271bx1434
druid:tv435zz8814
druid:tv465sy1765
druid:tw245jj3375
druid:tw895px4310
druid:tw957cv5461
druid:tx445hb6684
druid:tx869ph0160
druid:ty212cb6026
druid:ty485nd9645
druid:tz346yg1321
druid:tz787gy7414
druid:tz803mx7913
druid:vb361zf3845
druid:vb831hv7442
druid:vb993ng9670
druid:vc187ns9355
druid:vc497jv6018
druid:vc704wm8976
druid:vd370nv7524
druid:vd498pz2784
druid:vd745bn3644
druid:vd835mg5288
druid:vf961nb6005
druid:vg199nc6556
druid:vg324jr6593
druid:vg584sy5105
druid:vh535tr2948
druid:vh668bs5270
druid:vj026vt0170
druid:vj238jb7693
druid:vj506jn8471
druid:vj636zk2428
druid:vj807yp7042
druid:vj809by0153
druid:vj854pj3665
druid:vk042gy1460
druid:vk403jc9176
druid:vk510pf3953
druid:vk689kd5626
druid:vk709fh9739
druid:vm402ds8170
druid:vm464wn7735
druid:vm915kf6809
druid:vp058zq0823
druid:vp201ct4097
druid:vp244pn6067
druid:vp250hv0889
druid:vp352ww5653
druid:vq133td3705
druid:vq133yp6436
druid:vq184bb6328
druid:vq559bs7996
druid:vr842js0638
druid:vs514pn7419
druid:vs699hv3743
druid:vt675ww4508
druid:vt804vz4572
druid:vt831ft1731
druid:vt990qb4120
druid:vv061mq5514
druid:vv244ch9153
druid:vv399ht9423
druid:vv639rj3449
druid:vv767vq6547
druid:vx430fs2416
druid:vy083rm9574
druid:vy553qs2473
druid:vz523sh8261
druid:vz768dh8919
druid:wb209yb6723
druid:wb752tk3702
druid:wc512jh8609
druid:wc805hg0602
druid:wc866hj9833
druid:wd222vh1594
druid:wd254pq1461
druid:wd589rt4130
druid:wd709jg2788
druid:wf221jk4512
druid:wf272sg3629
druid:wf303mk1345
druid:wf365jc8106
druid:wf604fx5181
druid:wf626rr1198
druid:wg177fb2843
druid:wg393gc0308
druid:wg818cg0069
druid:wg915tv7780
druid:wh003jb8842
druid:wh010jd9237
druid:wh574wd8923
druid:wh733rj8799
druid:wh874fq3753
druid:wj002pm1906
druid:wj209mn0784
druid:wj805vm7745
druid:wj961xc5322
druid:wk532hp2565
druid:wk553qk6535
druid:wm187bm9859
druid:wm218qw0161
druid:wm363bg3776
druid:wm950qd0581
druid:wn028jq4056
druid:wn642xs9389
druid:wn970fv7310
druid:wq305vz9046
druid:wq787nw4639
druid:wr034gg3847
druid:wr181zn1136
druid:wr371xv2395
druid:wr479wx2990
druid:wr490xb9031
druid:wr533mm9727
druid:wr801yt6537
druid:wr819pk3945
druid:wr906xf9473
druid:ws600vv4636
druid:ws825rc9224
druid:ws931dn2984
druid:wt319hc2488
druid:wt673pw3518
druid:wv258ys5419
druid:wv428nb9083
druid:wv446yn2614
druid:ww476xd0644
druid:ww512nz8858
druid:ww637mw4018
druid:ww700dh5891
druid:ww820sf7419
druid:ww908xv4517
druid:ww916mk4487
druid:wx993kx8875
druid:wy225tv6192
druid:wy648vg8020
druid:wy875sg1530
druid:wz013fq8676
druid:wz110cw1514
druid:wz189pq6979
druid:wz654jy0779
druid:wz712tt1973
druid:wz903cs2466
druid:xb011bv1109
druid:xb920vn7102
druid:xc592bn8785
druid:xc677zk9683
druid:xc935bb3547
druid:xd350jv9965
druid:xd391hc5674
druid:xd499sm5592
druid:xd949qv5832
druid:xf208wk4472
druid:xf270cm2526
druid:xg441cx3781
druid:xg667gr4605
druid:xj264gy7650
druid:xj369zy2317
druid:xj391jk5611
druid:xj492xf3158
druid:xk494hq8901
druid:xk596tp3199
druid:xk684wc2476
druid:xk728gx8086
druid:xk886kn5384
druid:xk905tg0193
druid:xk906gg6034
druid:xm613qd8168
druid:xm841dh0967
druid:xm971jj1515
druid:xm972xb2956
druid:xn091qc4517
druid:xn249jp3210
druid:xn783yv1772
druid:xp203sz9316
druid:xp416kg1732
druid:xq620sh0830
druid:xq871ns4357
druid:xr031jq5952
druid:xr366xd9204
druid:xr535pz7637
druid:xr610jv8199
druid:xr640tm0255
druid:xr651kd7460
druid:xr925fh8985
druid:xs120yv1469
druid:xs167bb3967
druid:xv297tc5744
druid:xv774jj6151
druid:xv806zs5785
druid:xw319hw6113
druid:xw536qy6878
druid:xw700zs2408
druid:xw894gd1831
druid:xx214zq6843
druid:xy213tq3953
druid:xz089bc6565
druid:yb610xy6614
druid:yb711xv1062
druid:yb911tm3551
druid:yb924np3127
druid:yc472qr6472
druid:yc910qc3583
druid:yc983hm4138
druid:yd845wr7438
druid:yd922jw5114
druid:yf021bd0450
druid:yf084bb3073
druid:yf635vh9127
druid:yf722tn1369
druid:yg104cp6154
druid:yg401xx4163
druid:yg600qj1419
druid:yg794ff1074
druid:yh414ps0689
druid:yh710db2675
druid:yh711kg4756
druid:yh825nf6803
druid:yj725tc0909
druid:yj791yc8536
druid:yk203gf1929
druid:yk276mb1906
druid:yk460rh1053
druid:yk502ct8920
druid:yk739fb1298
druid:yk741kc8175
druid:yk890nf8632
druid:ym478vr7652
druid:ym733dy7285
druid:ym862db4014
druid:ym874yq2708
druid:yn069ws1192
druid:yn530ms1593
druid:yn566yz1577
druid:yn567rj7524
druid:yn576fx6668
druid:yn624nk6045
druid:yn784mv1663
druid:yn968vr7513
druid:yp170fr0129
druid:yp674mx8016
druid:yp861dj6649
druid:yq157wg8089
druid:yq897mx9896
druid:yr023pd8688
druid:yr496qp2566
druid:yr508cm2895
druid:yr891cx5883
druid:ys111hq8598
druid:ys781vp9990
druid:yt019js0079
druid:yt031dt0986
druid:yv090xk3108
druid:yv888dc9957
druid:yv920dp6586
druid:yw135pd1104
druid:yw668mh5468
druid:yx136qb8772
druid:yx763wc1075
druid:yy360cw7391
druid:yy435qt2161
druid:yy436vd6426
druid:yy958gs5961
druid:yz752hc2178
druid:yz756fd6804
druid:zb048qx3071
druid:zb386hj1820
druid:zb407mr1244
druid:zb676pt1805
druid:zb936wp1724
druid:zc108qs4757
druid:zc666vn6494
druid:zc822zs9181
druid:zd148dg8878
druid:zd164kw5923
druid:zd342hj7111
druid:zd587yp5802
druid:zf739wk1797
druid:zg038qp7973
druid:zg139hf3448
druid:zg684fd9646
druid:zg972cr7053
druid:zh036xt1541
druid:zh074rz0429
druid:zh266tg6451
druid:zh379qw9277
druid:zh718ty5611
druid:zh919dd9126
druid:zj032xr9642
druid:zj074rp6409
druid:zj494ys4628
druid:zk127ng8952
druid:zk135ry7430
druid:zk179qh8246
druid:zk481fh3026
druid:zk732rk5032
druid:zk738tt8654
druid:zk925wg4758
druid:zm124gt8240
druid:zm150hd4473
druid:zm788ph2453
druid:zn006ht3121
druid:zp007wk5160
druid:zp009gy6563
druid:zp189zv5402
druid:zp369wb8086
druid:zp944xt3979
druid:zq139hz6404
druid:zq147br5138
druid:zq714tc0381
druid:zr142bm8384
druid:zr371dm2747
druid:zr698sp2242
druid:zs166ht0272
druid:zs676js9011
druid:zs748bs9165
druid:zs976wd6410
druid:zs987xg2440
druid:zt371fb4152
druid:zt498cg6598
druid:zv034dd8328
druid:zv084kk5779
druid:zv303mp7402
druid:zv366hg8225
druid:zv541tb5122
druid:zv842kc7133
druid:zv906qr1644
druid:zv912gq3094
druid:zw673nh3928
druid:zw835tc1972
druid:zx392wj7796
druid:zx729jg0459
druid:zy062xv2633
druid:zz738fz4149
druid:zz987rw7807
```

# Skips due to item being in graveyard APO

```
druid:br953bm4680
druid:qf102ds5177
druid:sf729hj7119
druid:zw284gp9506
```

**Comment by mjgiarlo** [comment]
The above items skipped to already having a catkey are from the following collections:

```
druid:by600sn5927
druid:cb946pf0135
druid:gk894yk3598
druid:hf562ns6933
druid:hk912vy3606
druid:hn730ks3626
druid:nc035fh7626
druid:nc574ck0385
druid:nj253vv3687
druid:pg007dt6713
druid:pn808wc6253
druid:px088wv9627
druid:pz310tn8238
druid:qd259nz6640
druid:qq279mc0438
druid:qx111hs9166
druid:sq238mw3774
druid:tg242xv5347
druid:tn582bs2355
druid:wv537kq6898
druid:ww293jb1489
druid:xn104wc7286
druid:xx313vx3974
druid:zb231rn4458
```

**Comment by amyehodge** [comment]
I don't see anything here that gives me pause. @hannahfrost Do you think we need to check any individual items identified above from any of these collections?

druid:by600sn5927 - Electronic Acquisitions
druid:cb946pf0135 - Government Information State and Local Collection
druid:gk894yk3598 - Acquisitions Serials
druid:hf562ns6933 - Studies in Human-Thing Entanglement
druid:hk912vy3606 - Terman Engineering Library Supplemental Materials
druid:hn730ks3626 - Stanford University Libraries staff publications and research
druid:nc035fh7626 - Hopkins Marine Station of Stanford University Honors Theses
druid:nc574ck0385 - Government Information United States Federal Collection
druid:nj253vv3687 - Hoover Institution Library Materials
druid:pg007dt6713 - Government Information International Collection
druid:pn808wc6253 - Hopkins Marine Station Collection
druid:px088wv9627 - CoreLogic Tax and Deed Data
druid:pz310tn8238 - Master's Theses, Stanford Earth
druid:qd259nz6640 - Graduate School of Education International Comparative Education Master's Monographs
druid:qq279mc0438 - Dept. of Special Collections, Manuscripts Collections - Supplemental Materials
druid:qx111hs9166 - CoreLogic Loan Level Market Analysis (LLMA) Data
druid:sq238mw3774 - Digital Music Scores purchased by Music Library (Stanford-only access)
druid:tg242xv5347 - Non-governmental Organizations (NGOs) Collection
druid:tn582bs2355 - SSDS Data Annex
druid:wv537kq6898 - Free EEMs
druid:ww293jb1489 - GATT Digital Library Resources
druid:xn104wc7286 - Piano roll books
druid:xx313vx3974 - Stanford Theses and Dissertations
druid:zb231rn4458 - NewSpace Global Reports

**Comment by mjgiarlo** [comment]
@amyehodge Should I wait on this to begin importing on the H2 side?

**Comment by amyehodge** [comment]
@mjgiarlo Nope. Go ahead! We can make changes next time if @hannahfrost sees anything in here she thinks should still be migrated despite having a catkey. But I think that's highly unlikely.

**Comment by hannahfrost** [comment]
@mjgiarlo This list includes some things that I wasn't expecting. I am going to reach out to some of those collections owners about whether they need those items in H2 or not.

**Comment by amyehodge** [comment]
Please make sure that hydrus collections with the review workflow turned on continue to have the review workflow turned on when migrated to H2.

**Comment by mjgiarlo** [comment]
@amyehodge :speech_balloon: 
> Please make sure that hydrus collections with the review workflow turned on continue to have the review workflow turned on when migrated to H2.

New issue created for this: https://github.com/sul-dlss/happy-heron/issues/1240

**Comment by mjgiarlo** [comment]
# 39 contributor-related failures

```
Validation failed on druid:bc865wh0982, [] ["Contributors is invalid"]
Validation failed on druid:bj798mj3298, [] ["Contributors is invalid"]
Validation failed on druid:bx852hj9394, [] ["Contributors is invalid"]
Validation failed on druid:bz017bq7668, [] ["Contributors is invalid"]
Validation failed on druid:cd054gf2198, [] ["Contributors is invalid"]
Validation failed on druid:cr877ww8519, [] ["Contributors is invalid"]
Validation failed on druid:cv304vk9883, [] ["Contributors is invalid"]
Validation failed on druid:cy977mf7313, [] ["Contributors is invalid"]
Validation failed on druid:ft697cq3835, [] ["Contributors is invalid"]
Validation failed on druid:gp361hz5082, [] ["Contributors is invalid"]
Validation failed on druid:gw249gv9778, [] ["Contributors is invalid"]
Validation failed on druid:hn955wp7476, [] ["Contributors is invalid"]
Validation failed on druid:hp391tw8759, [] ["Contributors is invalid"]
Validation failed on druid:hw648fn9717, [] ["Contributors is invalid"]
Validation failed on druid:jr652rx2541, [] ["Contributors is invalid"]
Validation failed on druid:jw683dy3472, [] ["Contributors is invalid"]
Validation failed on druid:kn730cj8320, [] ["Contributors is invalid"]
Validation failed on druid:mp512fy0405, [] ["Contributors is invalid"]
Validation failed on druid:mr373pn0879, [] ["Contributors is invalid"]
Validation failed on druid:ng517gq2855, [] ["Contributors is invalid"]
Validation failed on druid:qm274fz0921, [] ["Contributors is invalid"]
Validation failed on druid:qm695ny4920, [] ["Contributors is invalid"]
Validation failed on druid:rh614yz5124, [] ["Contributors is invalid"]
Validation failed on druid:rt187sc7864, [] ["Contributors is invalid"]
Validation failed on druid:ry742gn4021, [] ["Contributors is invalid"]
Validation failed on druid:sv915zg2311, [] ["Contributors is invalid"]
Validation failed on druid:th866bg2433, [] ["Contributors is invalid"]
Validation failed on druid:th926tb9400, [] ["Contributors is invalid"]
Validation failed on druid:tj546dj4178, [] ["Contributors is invalid"]
Validation failed on druid:tk726kd4859, [] ["Contributors is invalid"]
Validation failed on druid:tm700dc0091, [] ["Contributors is invalid"]
Validation failed on druid:tp271xt6869, [] ["Contributors is invalid"]
Validation failed on druid:vh359gs8861, [] ["Contributors is invalid"]
Validation failed on druid:wp105cx4248, [] ["Contributors is invalid"]
Validation failed on druid:wx314xy1580, [] ["Contributors is invalid"]
Validation failed on druid:xn156wf1698, [] ["Contributors is invalid"]
Validation failed on druid:xq419cv5789, [] ["Contributors is invalid"]
Validation failed on druid:yj661fn2246, [] ["Contributors is invalid"]
Validation failed on druid:zb309sj9103, [] ["Contributors is invalid"]
```

**Comment by mjgiarlo** [comment]
@amyehodge all 3,825 works migrated just fine (with many of the same metadata warnings and placeholder behaviors as before re: contact emails and work types and abstracts). The above 39 items still have contributor-related problems, which caused them to have works without any work versions. This would be worth doing more digging into.

**Comment by mjgiarlo** [comment]
Closing for now since the run is done and the analysis has been written up.

**Comment by amyehodge** [comment]
The above contributor issues are all from the following three collections:

- Undergraduate Theses, School of Engineering
- Undergraduate Theses, Department of Physics
- Undergraduate Theses, Program in Engineering Physics

I looked at a few of these and am not immediately seeing anything.

**Comment by mjgiarlo** [comment]
@amyehodge OK, I think the problem is a failure to validate departments as contributors. I'll work on a PR!
